symphony/bdk/core/auth/exception.py

```diff
@@ -7,6 +7,7 @@ class AuthInitializationError(Exception):
     """
 
     def __init__(self, message: str):
+        Exception.__init__(self, message)
```

Comment:
Curious question: Why do you call the parent constructor for Exception ?
.
.
On internet forums, the usual answer would be `super.__init__` instead of `Exception.__init__` as it would avoid us using the base class name explicitly.

I however don't see the point of calling the parent constructor if we set the instance attributes. 
If you check the source code for Exception:
```Python
class Exception(BaseException):
    """ Common base class for all non-exit exceptions. """
    def __init__(self, *args, **kwargs): # real signature unknown
        pass
```
It only instanciates them for us. So my take would be droping the __init__ completly: https://docs.python.org/3.8/tutorial/errors.html -> 8.5. User-defined Exceptions

Reply:
I updated these exceptions following the discussion on this page: https://stackoverflow.com/questions/2901000/the-correct-way-to-define-an-exception-in-python-without-pylint-complaining/2901085#290108 . 
Pylint was complaining and raising warnings about it, so I followed this comment https://stackoverflow.com/a/2901085 and seems like Pylint is happy now. 

But if there's a better way on how to handle this I'm open to change it.

Reply:
These discussions seem a bit old (dates of 2010), since python 3+, you would see things more like this: `super().__init__`: https://stackoverflow.com/questions/1319615/proper-way-to-declare-custom-exceptions-in-modern-python

I explained how I see things and how it was documented in the official documentation. Calling the parent constructor is not doing anything anyways which I think we should remove it.
https://docs.python.org/3.8/tutorial/errors.html -> 8.5. User-defined Exceptions